shared-components/tl-date-time-card/tl-date-time-card-css.js

```diff
@@ -0,0 +1,17 @@
+import { css } from 'lit';
+
+export const dateTimeStyles = css`
+    .iconWrapper {
+        max-width: 13.4em;
+        min-height: 2.2em;
+        background: var(--primary-green);
+        border: 5px solid #1D0747;
+        border-radius: 4em;
```

Comment:
All sizes should be expressed in `rem` to provide a consistent (because it's related to `root`, not to the parent element)...